extensions/interactions/ItemSelectionInput/ItemSelectionInput.html

```diff
@@ -1,13 +1,55 @@
+<style>
+  .item-selection-form-container {
+    margin-top: -5px;
+  }
+  .item {
+    background: none;
+    border: none;
+    color: rgba(0,0,0,0.5);
+    margin: 0;
+    padding-left: 0;
+    padding-top: 9px;
+    text-align: left;
+    /* This is needed so that images stay bounded by the container in Firefox. */
+    width: 100%;
+  }
+  .item-container {
+    border-bottom: thin solid #EEEEEE;
+    margin-bottom: 5px;
+  }
+
+  .item-container md-input-group.md-default-theme label {
+    border-bottom: 0px;
+  }
+
+  .item:hover {
+    color: #00897B;
+  }
+</style>
+
 <script type="text/ng-template" id="interaction/ItemSelectionInput">
-  <form ng-submit="submitAnswer()">
-    <div class="checkbox" ng-repeat="item in items">
-      <label>
-        <input type="checkbox" ng-model="studentSelections[item]">
-        <span angular-html-bind="item"></span>
-      </label>
-    </div>
-    <md-button class="md-button-success" ng-click="submitAnswer()">Submit</md-button>
-  </form>
+  <div class="item-selection-form-container">
+    <form ng-submit="submitAnswer()">
+      <div class="item-container" ng-repeat="choice in choices">
+        <label ng-switch="isCheckbox">
+          <div ng-switch-when="true">
+            <md-checkbox class="item" ng-checked="answers.indexOf(choice) !== -1" ng-click="checkCount(choice)"
+              ng-disabled="maxed && answers.indexOf(choice) === -1" aria-label="choice">
+              <span angular-html-bind="choice"></span>
+            </md-checkbox>
+          </div>
+
+          <div ng-switch-when="false">
+            <button class="item" ng-click="submitAnswer(choice)" aria-label="choice">
+             <span angular-html-bind="choice"></span>
+            </button>
+          </div>
+        </label>
+      </div>
+      <md-button ng-if="isCheckbox" class="md-button-success"
+        ng-click="submitAnswer()" aria-label="Submit" ng-disabled="!min || !maxed">Submit</md-button>
```

Comment:
Perhaps there should be some sort of message asking the student to pick more choices if the submit button is disabled due to there not being enough.